script/bootstrap

```diff
@@ -60,6 +60,20 @@ setup_gitconfig () {
   fi
 }
 
+setup_git_symlinks () {
+  if ! [ $(readlink ~/.gitconfig) = */.dotfiles/developer/common/git/gitconfig.symlink ] &&
```

Comment:
You're using an assignment currently (`=`) instead of validating equality (`==`). You can also use the `-eq` operator (https://www.tldp.org/LDP/abs/html/comparison-ops.html).

Reply:
check!